installer/charts/rhtap-infrastructure/templates/odf/enable-odf-console-plugin.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.odf.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: patch-console-enable-odf
+  namespace: {{ .Values.odf.namespace }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    spec:
+      serviceAccountName: patch-console-sa
+      restartPolicy: OnFailure
+      containers:
+        - name: patch-console
+          image: quay.io/openshift/origin-cli:latest
+          command:
+            - /bin/bash
+            - -c
+            - |
+              echo "Patching Console resource to enable odf-console plugin..."
+              if oc get console.operator.openshift.io cluster -o json | grep -q '"odf-console"'; then
+                echo "odf-console plugin already enabled. Skipping patch."
+              else
+                oc patch console.operator.openshift.io cluster --type=json -p '[{"op":"add","path":"/spec/plugins/-","value":"odf-console"}]'
+              fi
```

Comment:
Add a final 'echo' statement so it's easy to see the success status from looking at the logs (eg `echo "# Success"`).

Reply:
Sure thing

Reply:
Done